src/style/mixins.scss

```diff
@@ -32,10 +32,12 @@
   }
 }
 
-@mixin iui-focus($color: var(--iui-color-foreground-primary), $offset: -1px, $thickness: 1px) {
+@mixin iui-focus($status: primary, $offset: -1px, $thickness: 1px) {
```

Comment:
This change is unrelated, please undo from this PR

Reply:
Reverted.